recipes/r-presenceabsence/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = '1.1.9' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-presenceabsence
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/PresenceAbsence_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/PresenceAbsence/PresenceAbsence_{{ version }}.tar.gz
+  sha256: 1a30b0a4317ea227d674ac873ab94f87f8326490304e5b08ad58953cdf23169f
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('PresenceAbsence')"           # [not win]
+    - "\"%R%\" -e \"library('PresenceAbsence')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=PresenceAbsence
+  license: MIT
```

Comment:
cran meta-data tells "unlimited"... is there any reference that it is also MIT?

Reply:
I was certain I saw it somewhere but I can't find it back, found the reference paper for this package though (https://www.jstatsoft.org/article/view/v023i11) it mentions GPL-2.0-or-later for the code, seems right ?

Reply:
Wondering if `LicenseRef-unlimited` would make sense?

Reply:
Maybe, I don't know if custom LicenseRef have any requirements to be used ? Like being defined somewhere in the tarball or online ?

Reply:
Updated it. That is all what is inside https://cran.r-project.org/src/contrib/PresenceAbsence_1.1.9.tar.gz ... linter seems to be happy.

Reply:
Ok wonderful ! Thanks :)